docs/best-practices.md

```diff
@@ -0,0 +1,28 @@
+Best Practices
+---------
+
+This page is a placeholder for best practices we recommend you to review while implementing Vue InstantSearch.
```

Comment:
Totally agree - I think that security info would be perfect to include in this section